src/library/features/analysis/analysisfeature.cpp

```diff
@@ -54,62 +52,74 @@ QVariant AnalysisFeature::title() {
     return m_Title;
 }
 
-QIcon AnalysisFeature::getIcon() {
-    return QIcon(":/images/library/ic_library_prepare.png");
+QString AnalysisFeature::getIconPath() {
+    return ":/images/library/ic_library_prepare.png";
 }
 
-void AnalysisFeature::bindWidget(WLibrary* libraryWidget,
-                                 KeyboardEventFilter* keyboard) {
-    m_pAnalysisView = new DlgAnalysis(libraryWidget,
-                                      m_pConfig,
-                                      m_pTrackCollection);
-    connect(m_pAnalysisView, SIGNAL(loadTrack(TrackPointer)),
-            this, SIGNAL(loadTrack(TrackPointer)));
-    connect(m_pAnalysisView, SIGNAL(loadTrackToPlayer(TrackPointer, QString)),
-            this, SIGNAL(loadTrackToPlayer(TrackPointer, QString)));
-    connect(m_pAnalysisView, SIGNAL(analyzeTracks(QList<TrackId>)),
-            this, SLOT(analyzeTracks(QList<TrackId>)));
-    connect(m_pAnalysisView, SIGNAL(stopAnalysis()),
-            this, SLOT(stopAnalysis()));
+QString AnalysisFeature::getSettingsName() const {
+    return "AnalysisFeature";
+}
 
-    connect(m_pAnalysisView, SIGNAL(trackSelected(TrackPointer)),
-            this, SIGNAL(trackSelected(TrackPointer)));
+QWidget* AnalysisFeature::createPaneWidget(KeyboardEventFilter*, int paneId) {
+    WTrackTableView* pTable = createTableWidget(paneId);
```

Comment:
Oh yes I remember now, since this function is inherited multiple times, it's difficult to control which will be the top container returned, this is because when calling `createPaneWidget` there are multiple functions inherited and it's really easy to set the parent after calling the function than before in some cases. 

If you don't like it I'll take a look and change the order of widget creation to do not look as weird as it looks.